app/src/main/java/org/sopt/sample/presentation/music/MusicFragment.kt

```diff
@@ -0,0 +1,50 @@
+package org.sopt.sample.presentation.music
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.viewModels
+import androidx.recyclerview.widget.LinearLayoutManager
+import dagger.hilt.android.AndroidEntryPoint
+import org.sopt.sample.R
+import org.sopt.sample.base.BindingFragment
+import org.sopt.sample.databinding.FragmentMusicBinding
+import org.sopt.sample.presentation.music.adapter.MusicListAdapter
+import org.sopt.sample.presentation.music.data.MusicAdd
+import org.sopt.sample.presentation.state.UiState
+
+@AndroidEntryPoint
+class MusicFragment : BindingFragment<FragmentMusicBinding>(R.layout.fragment_music) {
+    private val viewModel: MusicViewModel by viewModels()
```

Comment:
```suggestion
    private val viewModel: MusicViewModel by activityViewModels()
```

홈프래그먼트와 뮤직 다이얼로그에서 뷰모델 초기화 방식을 by activityViewModels()로 바꿔주시면 발생하신 이슈가 해결될 것 같습니다~ by activityViewModels()는 Fragment에서만 사용 가능한 viewModel 초기화 방식입니다!! Fragment가 생성된 Activity의 라이프사이클에 ViewModel을 종속시키기 때문에 같은 Activity를 공유하는 Fragmenet간 데이터 공유가 가능하게 됩니다~
현재는 홈 프래그먼트와 뮤직 다이얼로그가 각각 viewModels()를 사용하고 있고 뮤직 다이얼로그에서 musicList에 값을 setValue하더라도 뮤직 다이얼로그(프래그먼트) 라이프사이클에 뷰모델을 종속시켜서 다이얼로그가 종료되면 setValue한 값도 사라지게 되기 때문이죠!